var/spack/repos/builtin/packages/dealii/package.py

```diff
@@ -68,13 +68,20 @@ class Dealii(CMakePackage):
             description='Compile with Python bindings')
     variant('int64',    default=False,
             description='Compile with 64 bit indices support')
+    variant('optflags', default=False,
+            description='Compile using additional optimization flags')
 
     # required dependencies, light version
     depends_on("blas")
     # Boost 1.58 is blacklisted, see
     # https://github.com/dealii/dealii/issues/1591
     # Require at least 1.59
     # +python won't affect @:8.4.2
+    # FIXME: once conretizer can unite unconditional and
```

Comment:
typo: conretizer -> concretizer

P.S. Does this not actually work? I was going to use some logic like this in #3210. Haven't tested it yet though.